elements/lisk-p2p/src/p2p.ts

```diff
@@ -390,6 +394,18 @@ export class P2P extends EventEmitter {
 			outboundShuffleInterval: config.outboundShuffleInterval
 				? config.outboundShuffleInterval
 				: DEFAULT_OUTBOUND_SHUFFLE_INTERVAL,
+			latencyProtectionRatio:
+				typeof config.latencyProtectionRatio === 'number'
```

Comment:
Do we really need to check this? I think the typing should enforce it by itself.

Reply:
It's necessary because say a user enters `0`. In this case `latencyProtectionRatio` is evaluated to false which would default to `0.068`. Instead if users enters `0` I want to disable latency protection.